### ISSUE TITLE
Element type is invalid

### ISSUE THREAD

**Issue by ErkanSensei** [react-native-community/react-native-svg#504]
I keep getting this error:

<img width="545" alt="screen shot 2017-11-10 at 4 41 28 pm" src="https://user-images.githubusercontent.com/17013914/32681649-20bfe324-c636-11e7-8914-27b573c02c12.png">

I've tried multiple combos of RN, React, and RNSVG; nothing seems to work. I've linked, unlinked, linked again, rebuilt, cleaned; everything I could think of.

Current RN Version: 0.50.0
Current React Version: 16.0.0
Current RNSVG Version: 6.0.0

**Comment by jnberry** [comment]
I see the same, with the same versions.

**Comment by msand** [comment]
Definitely looks like the linking failed. Try linking manually.

**Comment by ErkanSensei** [comment]
So what ended up happening is that I had migrated my project from Expo to vanilla RN. What I didn't know is that the API is slightly different for Expo's implementation of RNSVG.

For RNSVG you simply import elements like:

`import SVG, { Circle, Rect } from 'react-native-svg'` 

whereas in Expo it's:
`import { Svg } from 'expo'
<Svg.Circle />
`
Once I imported the elements from RNSVG properly, it started working. I actually ended up making a website that helps convert SVGs to RNSVG/Expo properly after this issue 😛 
http://erkansensei.com/SVG-to-ReactNativeSVG/
